Description: Fix FTBFS.
 Workaround automake sanity check that would prevent us from
 installing lib/* into lib/backupninja/.
Origin: upstream, https://labs.riseup.net/code/projects/backupninja/repository/revisions/0a0fbb46baa1fd5e6bb68c9e3f9f0b28dfa17e15
Debian-Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=660617

diff --git a/lib/Makefile.am b/lib/Makefile.am
index 11eba37..9a54736 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -1,6 +1,7 @@
-pkglib_SCRIPTS = easydialog parseini tools vserver
+my_execbindir = $(pkglibdir)
+my_execbin_SCRIPTS = easydialog parseini tools vserver
 
-CLEANFILES = $(pkglib_SCRIPTS)
+CLEANFILES = $(my_execbin_SCRIPTS)
 
 EXTRA_DIST = easydialog.in parseini.in tools.in vserver.in
 
